modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID array using the `mdadm` utility, and integrate the RAID array with {lvms}.
```

Comment:
Expand RAID on first occurance (except the title as it can make the title long):

redundant array of independent disks (RAID).

IBM SG :

![Screenshot from 2024-01-31 11-13-40](https://github.com/openshift/openshift-docs/assets/57954076/cee38e65-8b0d-4da0-bcbe-7677277f5ea2)

Safer to spell out first occurence than find if a term is wide nomenclature across industry.

Reply:
Updated as per the suggestion.